cluster/gce/config-default.sh

```diff
@@ -255,15 +255,7 @@ if [[ (( "${KUBE_FEATURE_GATES:-}" == *"AllAlpha=true"* ) || ( "${KUBE_FEATURE_G
   RUN_CONTROLLERS="${RUN_CONTROLLERS:-*,endpointslice}"
 fi
 
-# Optional: set feature gates
```

Comment:
i think we should leave this in!

Reply:
`FEATURE_GATES="${KUBE_FEATURE_GATES:-}"`

Reply:
I kept it in initially but `pull-kubernetes-verify` job failed because `FEATURE_GATES` was unused and therefore had to remove it. Here is the error message: 
```
In ./cluster/gce/config-test.sh line 160:
FEATURE_GATES=${KUBE_FEATURE_GATES:-}
^-----------^ SC2034: FEATURE_GATES appears unused. Verify use (or export if used externally).
In ./cluster/gce/config-default.sh line 259:
FEATURE_GATES="${KUBE_FEATURE_GATES:-}"
^-----------^ SC2034: FEATURE_GATES appears unused. Verify use (or export if used externally).
For more information:
  https://www.shellcheck.net/wiki/SC2034 -- FEATURE_GATES appears unused.
```
This is also captured in the comment above: https://github.com/kubernetes/kubernetes/pull/112980#issuecomment-1289968193.

Reply:
@swatisehgal it's used in `config-helper.sh`.

please disable shell check for this line. see examples here:
https://cs.k8s.io/?q=SC2034&i=nope&files=&excludeFiles=&repos=

Reply:
Fixed, Thanks.